src/offset/utc.rs

```diff
@@ -4,11 +4,14 @@
 //! The UTC (Coordinated Universal Time) time zone.
 
 use std::fmt;
+#[cfg(feature="local")]
```

Comment:
No, this is correct. With the local feature enabled, we have `extern crate time as oldtime` in the crate root. This is needed to get the current time. With `UTC::today` and `UTC::now` cfged-out, this import is not used.